src/js/services/apiservice.js

```diff
@@ -0,0 +1,19 @@
+export const RegisterUser = (username, surname, passsword, email, phoneNumber, membershipType)=>{
+
```

Comment:
as mentioned previously might be easier to send through this.state and then unwrap it in the function like const {username, surname, passsword, email, phoneNumber, membershipType} = this.state